kratos/utilities/math_utils.h

```diff
@@ -357,6 +351,28 @@ class MathUtils
             KRATOS_ERROR << "Size not implemented. Size: " << TDim << std::endl;
         }
 
+        //Find the condition number to define is inverse is OK
+        double input_matrix_norm = 0.0;
+        double inverted_matrix_norm = 0.0;
+        for(unsigned int i = 0; i < TDim; i++)
+        {
+            for(unsigned int j = 0; j < TDim; j++)
+            {
+                input_matrix_norm += std::pow(InputMatrix(i,j),2);
+                inverted_matrix_norm += std::pow(InvertedMatrix(i,j),2);
```

Comment:
Is this method not yet implemented somewhere else?

Reply:
Yes, but for sparse matrices

Reply:
wow :)
The entire Kratos does not have an implementation for the L2-Norm? :D

Reply:
This way the double loop is done only once